### ISSUE TITLE
A11y_.NET Core_WPF_CustomComboBox_MainWindow_movieSelected_ScreenReader: The screen reader is not announcing the selected movie when the focus is on "Watch Now".

### ISSUE THREAD

**Issue by Truptik12** [dotnet/wpf#6738]
[6738.zip](https://github.com/dotnet/wpf/files/9008587/6738.zip)
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yTCS;#.NET Core[WPF]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG4.1.2;#DesktopApp;#Win11;#Narrator;#NVDA;#Benchmark;

### Environment Details:
.NET Core
WPF
Operating System: Windows 11 Enterprise 21H2

### Steps to Reproduce:

1. Launch VS 2019 Int Preview
2. Navigate to Sample Applications and right click on the  Custom Combo box  and click on build.
3. Then right click on Custom Combo box then Click on Debug-->Start New Instance. 
4. Main Window screen should open.
5. Navigate to accordion and select any movie by clicking the button present to watch now.
6. Turn on the screen reader and observe the announcement.

### Actual:
The screen reader is not announcing the selected movie when the focus is on "Watch Now".

### Expected:
The screen reader should announce the selected movie when the focus is on "Watch Now".

### User Impact:
Screen reader users wont know which movie is selected.

**Comment by anjalisheel-wpf** [comment]
https://user-images.githubusercontent.com/85492722/182108403-47ab4014-9693-4877-8274-56bb4f863051.mov

This is working as expected, hence closing it.

**Comment by anjalisheel-wpf** [comment]
@Truptik12 Please test and confirm.

**Comment by Truptik12** [comment]
Closing the bug as non blocking issue
